code/modules/clothing/rogueclothes/armor/blacksteel.dm

```diff
@@ -53,3 +53,23 @@
 	smeltresult = /obj/item/ingot/blacksteel
 	armor_class = ARMOR_CLASS_MEDIUM
 	smelt_bar_num = 2
+
+/obj/item/clothing/suit/roguetown/armor/plate/blacksteel_halfplate
+	slot_flags = ITEM_SLOT_ARMOR
+	name = "blacksteel half-plate"
+	desc = "A sturdy set of plate armor protecting only the chest and groin, smithed of rare blacksteel. This prestigious set of armor has a set of pauldrons."
+	body_parts_covered = CHEST|GROIN|VITALS
```

Comment:
Maybe give it arm protection 'cause of pauldrons? Otherwise it doesn't seem like much of an upgrade over the existing blacksteel cuirass.